seatunnel-examples/seatunnel-engine-examples/src/main/resources/examples/fake_to_console.conf

```diff
@@ -30,14 +30,24 @@ source {
   # This is a example source plugin **only for test and demonstrate the feature source plugin**
   FakeSource {
     result_table_name = "fake"
-    field_name = "name,age",
-    parallelism = 2
+    parallelism = 1
+    schema = {
+      fields {
+        name = "string"
+        age = "int"
+      }
+    }
   }
 
   FakeSource {
     result_table_name = "fake"
```

Comment:
duplicate result_table_name?

Reply:
This is a feature that multi-source to one sink use same result_table_name.